src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -35,10 +39,18 @@ type Props = {
     feedItems?: FeedItems,
 };
 
+type Props = {
+    logger: LoggerType,
```

Comment:
Since most people won't care about the logger props could we do `propsWithContext` and leave props alone?